frontend/src/config/config.ts

```diff
@@ -0,0 +1,17 @@
+import { Config } from '../constants/datatypes'
+
+export let CONFIG: Config
+
+try {
+    // tslint:disable:no-var-requires
+    CONFIG = require('../../../shared/config.json')
+} catch (e) {
+    // tslint:disable:no-var-requires
+    CONFIG = require('../../../shared/config.json')
```

Comment:
? try this and if it doesn't work just try again?

Reply:
Couldnt get it to work yet but the idea is to try the docker-compose path and the local path